README.md

```diff
@@ -1,10 +1,10 @@
 ![alt text](https://github.com/bitDubai/media-kit/blob/master/MediaKit/Fermat%20Branding/Fermat%20Logotype/Fermat_Logo_3D.png "Fermat Logo")
 
-# Learning Fermat
+# Getting to know Fermat
```

Comment:
@daveo90 mmm..getting to know it is not the same as learning...learning is more organized and requires more commitment from the subject who is learning...please ask @bitDubai/TatianaMoroz about it ( she is the only english native speaker in Fermat so far..)